docs/current_docs/cookbook/cookbook.mdx

```diff
@@ -1821,6 +1821,33 @@ Build from a Dockerfile with a mounted secret from the host environment:
     </TabItem>
     </Tabs>
 
+### Customize secret caching behavior
+
+By default, the cache for operations that include a secret will be based on the plaintext value of the secret. Operations that include secrets with the same plaintext value may share cache, but if the plaintext differs then the operation will not share cache.
```

Comment:
```suggestion
By default, the cache for operations that include a secret will be based on the plaintext value of the secret. Operations that include secrets with the same plaintext value may share the cache, but if the plaintext differs then the operation will not share the cache.
```

Question: this refers to "cache for operations" In our docs, we use the terms "layer cache" and "cache volumes": https://docs.dagger.io/features/caching. Which of these is the "cache for operations"? Can we replace "cache for operations" with one of the other two terms, so readers understand can grok it better?

Reply:
I updated "share cache" to be "share layer cache entries" ("share the layer cache" didn't sound right to me)